nl/mail/chrome/messenger/updates/pbUpdates.dtd

```diff
@@ -1,17 +1,17 @@
 <!ENTITY releaseNotes.label "Release Notes:">
-<!ENTITY cancelButton.label "Cancel">
-<!ENTITY installUpdateButton.label "Install Update">
-<!ENTITY newVersion.label "A new version of Postbox is available!">
+<!ENTITY cancelButton.label "Annuleren">
+<!ENTITY installUpdateButton.label "Installeer Update">
+<!ENTITY newVersion.label "Er is een nieuwe versie van Postbox beschikbaar">
```

Comment:
CORRECTION NEEDED, it should be: "Er is een nieuwe versie van Postbox beschikbaar!"
- This is if we want to keep exclamation marks in translations.